### ISSUE TITLE
App crashes when transmitting selection from Row data plot to Row statistics table

### ISSUE THREAD

**Issue by csoneson** [csoneson/iSEE#286]
This came up while teaching today (using current `master`):
- Launch allen data app
- Select some points in Row data plot 1
- Receive selection from Row data plot 1 in Row statistics table 1
-> App crashes, with a 
```
Warning: Error in : subscript contains invalid names
  [No stack trace available]
```
Happening with other data sets as well (but worked for me ~a month ago). Will investigate further tomorrow.

**Comment by LTLA** [comment]
Any progress?

**Comment by csoneson** [comment]
Not really, in the end I didn't have so much time as I had hoped today. It's definitely happening while transmitting the selection, and happens both with saved and active selections. Just came home from Cambridge, hopefully I'll find some time between meetings tomorrow to figure out what's going on.

**Comment by LTLA** [comment]
This should be resolved upon merge of #285.

**Comment by csoneson** [comment]
Great. I tried `prerelease` and indeed it seems to work.
